shared_workflow/install_shared.py

```diff
@@ -98,7 +98,9 @@ def install_simulation(
     }
 
     # VM params
-    vm_params_dict = utils.load_yaml(vm_params_path)
+    vm_params_dict = utils.load_yaml(
```

Comment:
Are we using the vm_params path variable anymore?

Reply:
line 112 
https://github.com/ucgmsim/slurm_gm_workflow/blob/392a9f80bd68c5852a89175d86739447b6539ffc/shared_workflow/install_shared.py#L112
still use it. 
the parent script that calls this function still pass this as a argument ( and is incorrect one in the test, using params_vel instead of the newest config format)
So there is two options:
1. remove the logic from parent script, so the config is always read from vel_mod_dir
    less flexible, but stable
2. leave the logic in parent script and remove it from this scripts. whoever calls install script should handle the path correctly.
    more flexible, but prone to mismatch crashes

Reply:
the vm_params_path in SimParams are not being used in install either, its serves as recording purpose.
so it should be re-directed to the value/choice whatever we decided to use from the two option i mentioned.

Reply:
We can just refactor vm_params_path out and replace it with sim_struct.get_vm_yaml(vm_dir).
"Mr Huang, tear down this parameter!"

Reply:
Yeah. VM params is defined by the simulation structure. If that moves, then we have other issues.